services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: None = None,
+        actors_ids: None = None
+) -> None:

```

Comment:
The return type should be `QuerySet` instead of `None` since the function returns a queryset of movies.